Airports/EGPE/Ownership.txt

```diff
@@ -16,10 +16,26 @@ BORDER:PETWR
 ARRAPT:EGPE
```

Comment:
Remove all instances of PEA in this file, as we now only define the Radar position.

Line 2 has an instance of AET which should be PET